src/components/MVP/Header/Header.test.tsx

```diff
@@ -112,7 +112,7 @@ describe('Header component', () => {
 
   describe('on the news page', () => {
     beforeEach(() => {
-      mockedUseRouter.mockReturnValueOnce({
+      mockedUseRouter.mockReturnValue({
```

Comment:
This has to be returned multiple times since `useRouter` now is called on both the Header & NavLink components